gobblin-api/src/main/java/org/apache/gobblin/service/ServiceConfigKeys.java

```diff
@@ -58,6 +58,10 @@
   public static final String GOBBLIN_SERVICE_FLOW_CATALOG_LOCAL_COMMIT = GOBBLIN_SERVICE_PREFIX + "flowCatalog.localCommit";
   public static final boolean DEFAULT_GOBBLIN_SERVICE_FLOW_CATALOG_LOCAL_COMMIT = true;
 
+  // Job Level Keys
+  public static final String WORK_UNIT_BYTE_SIZE = GOBBLIN_SERVICE_PREFIX + ".work.unit.byte.size";
```

Comment:
WORK_UNIT_BYTE_SIZE is confusing to read. Can we just call it work_unit_size? Also, is this config only intended for File-based sources? How about other sources e.g. Kafka Source, where this config could be used to mean number of records?

Reply:
This isn't meant for only file-based sources. It will also be used for record-based sources so it will make more sense to call it `work_unit_size` like you're suggesting.